### PR TITLE
Readme rework

### PR DESCRIPTION
**Pull request by lightspeedana**
So uh I decided to rework readme.md, partially due to some apparently confusing guidelines and to make the usage of meta clearer to members new to the server/GH; let me know if there's anything I need to change or add, as I'm sure this isn't entirely complete itself.

@thebeanogamer @teamshortcut I leave this in your hands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,29 @@
-# meta
+# CD Discord Meta Repository
```
Comment (thebeanogamer) [inline README.md]:
```suggestion
# Cyber Discovery Community Discord Meta Repository
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
 
+## Code of Conduct
+This repository is for serious suggestions to the server - we expect all suggestions and discussion to be made respectfully.
+
+We welcome differing opinions and debate, but please make sure that this is done inoffensively. Be welcoming, inclusive, respectful, empathetic and constructive, and the whole process will be much smoother. 😄
+
 ## Usage
-If you have any problems with the server, or aspects you think should be changed, make an Issue! For any suggestions, improvements, or solutions to current Issues, just make a Pull Request. Discussion can then occur in this repository, among server members and admins. When making a PR, you may find it helpful to refer to or edit some of the server files, such as [CHANNELS.md](/CHANNELS.md), [ROLES.md](/ROLES.md) or for other suggestions (e.g. emojis or more general changes), [MISCELLANEOUS.md](/MISCELLANEOUS.md). After a PR has been approved, it will go into the Discord server to be voted on, to assess its popularity before potentially being implemented.
+We understand the meta repo can be somewhat confusing, so here's exactly how to use it:
```
Comment (thebeanogamer) [inline README.md]:
```suggestion
We understand that the meta repo can be somewhat confusing, so here's exactly how to use it:
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+### Making Issues
+If you have any problems with the server, or aspects you think should be changed, make an Issue! Make sure no one has addressed the issue you have in another issue or PR before making it - if not, you can do the latter by:
+1. Navigate to the `Issues` tab.
+2. Click on the `New issue` button in the top-right corner. 
+3. Follow the template prompt and describe the issue. 
+Following comments and actions by the Admin Team or server members on the issue, one can then make a PR if a consensus has been achieved or the issue seems popular. 
+
+### Making PRs
+For any suggestions, improvements, or solutions to current Issues, just make a Pull Request. Follow these steps:
+1. Either fork the repository or make edits to files on a separate branch in `meta`.
```
Comment (thebeanogamer) [inline README.md]: Only we can create branches, all other users need to use forks.

**Review by thebeanogamer — CHANGES_REQUESTED** [review]

**Review by lightspeedana — COMMENTED** [review]
@thebeanogamer made changes as per.

**Comment by Sh3llcod3** [comment]
I am so sorry, I still need to fix `:readme pull` not working, let me just spin up my dev machine. :man_facepalming:

**Review by teamshortcut — APPROVED** [review]

**Review by thebeanogamer — APPROVED** [review]

**Pull request merged**